pulsar/resource_pulsar_topic.go

```diff
@@ -86,6 +86,23 @@ func resourcePulsarTopic() *schema.Resource {
 					},
 				},
 			},
+			"retention_policies": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"retention_minutes": {
```

Comment:
```suggestion
						"retention_time_minutes": {
```
Here seems to be a small name problem in both topic and namespace.

Reply:
It is same with `namespace`.

Reply:
Yes, the ones in `namespace` should be wrong too.

Reply:
I knew how to fix this, thanks.